test/built-ins/Set/prototype/union/union-function-properties.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2023 Anthony Frehner. All rights reserved.
```

Comment:
The name of this file could be clarified: I think we generally use just `length.js`